examples/bert/user_script.py

```diff
@@ -160,19 +161,19 @@ def __getitem__(self, index):
         return input_dict, label
 
 
-def inc_glue_calibration_reader(data_dir, batch_size, *args, **kwargs):
-    bert_dataset = BertDataset("Intel/bert-base-uncased-mrpc")
-    bert_dataset = IncBertDataset(bert_dataset.get_eval_dataset())
-    return DefaultDataLoader(dataset=bert_dataset, batch_size=batch_size)
+def inc_glue_calibration_reader(data_dir, batch_size, **kwargs):
+    dataset = BertDataset("Intel/bert-base-uncased-mrpc")
+    dataset = IncBertDataset(dataset.get_eval_dataset())
+    return DefaultDataLoader(dataset=dataset, batch_size=batch_size)
 
 
 # -------------------------------------------------------------------------
 # Accuracy Calculation Function
 # -------------------------------------------------------------------------
 
 
-def eval_accuracy(model: OliveModelHandler, data_dir, batch_size, device, execution_providers):
-    dataloader = create_dataloader(data_dir, batch_size)
+def eval_accuracy(model: OliveModelHandler, device, execution_providers, data_dir, batch_size):
```

Comment:
this function has `data_dir` and `batch_size` as additional args but the configs only pass `batch_size`. Also, looks like `data_dir` is not used correctly. From the original implementation https://github.com/microsoft/Olive/blame/68978f242b3fd8955d584648693a4f4c7cf8fa13/examples/bert/user_script.py#L149, Looks like we can just remove `data_dir` arg. and create the `dataloader` directly

Reply:
Done!